seed_files/scripts/get-factionkills.sh

```diff
@@ -0,0 +1,35 @@
+#/bin/bash
+
+# get the csv output from the spreadheet and 
+# strip the first column and first row
+# then remove duplicates (there will be many) 
+echo Getting the sheet data. it will take ages.
+
+wget --output-file=wget.log  --output-document=- 'https://docs.google.com/spreadsheets/d/e/2PACX-1vR8J04FrEoJ_a4nPI6rIyIsH2X41iQmICUaxK_SwdvL8PVq1-CBDynCF1vo3oFNM5krjwotkaIrH5Al/pub?gid=1871731906&single=true&output=csv' | cut -d"," -f1-9 | tail -n +2  > /tmp/factionkills.csv
+
+wget --output-file=wget.log  --output-document=- 'https://docs.google.com/spreadsheets/d/e/2PACX-1vSNTBQY-VlPzT9O6qLaKeC8Fgfhnbbca_BFm7koYJZsJBPLLGvIu88SQZeonqTIIeC-LYRcAywXpabY/pub?gid=1871731906&single=true&output=csv' | cut -d"," -f1-9 | tail -n +2  >> /tmp/factionkills.csv
+
+# damnit google why don't you put the final \r at the end!
+dos2unix /tmp/factionkills.csv
+
+# load that sucka!
+echo Database Root Password
+mysql -u root -p << endsql
+use capiv2;
```

Comment:
Might I recommend that an input variable is passed while running the script to use a specific database, I believe with bash you can use `$1` for the "first" variable passed so something like `./get-factionkills.sh capiv2` will use that specific database.

As an alternative we can default to `capiv2` if empty. Will make it easier as typically I use `capiv2_dev`, `capiv2_base`, and `capiv2_example`